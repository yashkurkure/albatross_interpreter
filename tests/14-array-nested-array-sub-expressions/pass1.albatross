arr1[arr2[0]] := arr3[arr4[10+variable] - arr5[arr0[0]]];