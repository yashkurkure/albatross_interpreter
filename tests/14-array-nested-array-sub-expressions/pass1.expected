arr3[(arr4[((10)+(variable))]-arr5[arr0[(0)]])]

